idc/metadata_utils.py

```diff
@@ -69,15 +80,16 @@ def get_collex_metadata(filters, fields, with_docs=True):
 
     solr_result = query_solr_and_format_result({
         'collection': 'tcga_clin_bios',
-        'fields': None,
+        'fields': fields,
         'fqs': query_set,
         'query_string': "*:*",
         'facets': solr_facets,
-        'limit': 0,
+        'limit': record_limit,
         'collapse_on': 'case_barcode',
-        'counts_only': True
+        'counts_only': False
     })
```

Comment:
This really needs to remain True (and limit to 0). We DON'T want to fetch clinical records at all, just counts. Records are only fetched from the 'images' collections.

Reply:
I think we will need clinical info (age, vital status) for the Patient table we talked about at the meeting this morning.

Reply:
So I think per Bill we're not supplying any clinical data, in which case this change can be reverted and we can push this through.